koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -2282,6 +2283,13 @@ def _execute_processing_script(self, script_file_path, sql_params):
             if sql_stmt:
                 sql_stmt, params = self.jinja_sql.prepare_query(sql_stmt, sql_params)
                 with connection.cursor() as cur:
+                    try:
+                        LOG.debug(cur.mogrify(sql_stmt, params).decode("utf-8"))
+                    except ProgrammingError:
+                        LOG.error(f"STATEMENT: {sql_stmt}")
+                        LOG.error(f"PARAMS: {params}")
+                        LOG.error(f"INPUT_PARAMS: {sql_params}")
```

Comment:
Let's consolidate these into a single Log.error with a meaningful error message. Otherwise we will get 3 different sentry alerts with unclear information.
```
LOG.error(f"something happened! STATEMENT: {sql_stmt} | PARAMS: {params} | INPUT_PARAMS: {sql_params}")
```